doc/api/crypto.md

```diff
@@ -622,7 +622,7 @@ provided, `otherPublicKey` is expected to be a [`Buffer`][],
 If `outputEncoding` is given a string is returned; otherwise, a
 [`Buffer`][] is returned.
 
-### diffieHellman.generateKeys([encoding])
+### diffieHellman.generateKeys(\[encoding\])
```

Comment:
This escaping of `[encoding]` is the only functional difference that this rule caught. Before it was adding a hyperlink in the tile because that bracket work is used as a reference link i this document